app/PilotPassport.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\HasMany;
+
+class PilotPassport extends Model {
+    protected $table = "pilot_passport";
+
+    public static function airfieldInPilotChallenge($airfield, $cid): bool {
+        $enrollments = PilotPassportEnrollment::where('cid', $cid)->get();
+        if ($enrollments->isEmpty()) {
+            return false;
+        }
```

Comment:
We can take this out. If it's empty, the next loop won't run at all and will return false at the end of the function

Reply:
Good point - done